features/FxA-83-sign-in-confirmation-email/README.md

```diff
@@ -222,28 +222,57 @@ against unverified tokens.
 
 ## Results
 
-Sign-in confirmation was deployed over several train cycles.
-The feature was turned on at 6-24-2016 for 2% of our users and
-gradually increased to 100%. During the deployment we monitored
-the sign-in success and error rates to ensure that users were
-not negatively impacted.
+Sign-in confirmation was deployed over several
+train cycles. The feature was turned on at 6-24-2016
+for 2% of our users and gradually increased to 100%.
+During the deployment we monitored the sign-in success
+and error rates to ensure that users were not negatively
+impacted.
 
 To help gauge the success of the feature, several dashboards
 were created.
 
 ### Sync sign-in success rate
 
-Pre sign-in confirmation metrics show that we had between a 42% to 52%
-sync sign-in success rate. From [first dashboard](https://app.datadoghq.com/dash/163668/fxa-content---signin-confirmation), with sign-in confirmation, we have maintained this sign-in rate.
+Pre sign-in confirmation metrics show that we had
+between a 42% to 52% sync sign-in success rate.
```

Comment:
+1 to what @shane-tomlinson said; my high-level thinking is basically that we want an explicit "you completed what you were trying to do" event that's emitting at the very last stage of the flow.  For sync signins that's when the browser actually succeeds in connecting to sync.  For oauth signins that's when the relier successfully obtains their access token.  Neither of those can be measured using only client-side metrics.

IOW, bring on the `flow.completed` flow event for future explorations of this type :-)